packages/library-utils/src/lib.rs

```diff
@@ -74,24 +76,67 @@ impl GetId for LibraryAccountType {
             LibraryAccountType::LibraryId(id) => *id,
         }
     }
+
+    fn get_account_id(&self) -> Id {
+        match self {
+            LibraryAccountType::Addr(_) => {
+                panic!("LibraryAccountType is an address")
+            }
+            LibraryAccountType::AccountId(id) => *id,
+            LibraryAccountType::LibraryId(_) => panic!("LibraryAccountType is a library id"),
+        }
+    }
+
+    fn get_library_id(&self) -> Id {
+        match self {
+            LibraryAccountType::Addr(_) => {
+                panic!("LibraryAccountType is an address")
+            }
+            LibraryAccountType::AccountId(_) => panic!("LibraryAccountType is a account id"),
+            LibraryAccountType::LibraryId(id) => *id,
+        }
+    }
 }
 
 impl GetId for u64 {
     fn get_id(&self) -> Id {
         *self
     }
+
+    fn get_account_id(&self) -> Id {
+        panic!("u64 has not account")
```

Comment:
I removed the `get_id` method, and left the other 2, the tl;dr of this is we want to catch some errors where might get confused between an account id and library id.

This is a little more complex because of how we use ids in the manager, ids are either a number, or our custom `LibraryAccountType` and there are 2 ways we use them:
1. If we need a type that can either be a number or an address, we use `LibraryAccountType`
2. If we want just the id, we can just use numbers. (like the links in the config)

We have the `GetId` trait to basically have a single type that we can use where we need an id, for example in `get_account` method, we don't care what type of id you pass us, as long as it implements `GetId`, so its up to the caller to choose what he pass which varies based on different scenarios.
If the caller pass a number, we assume the caller knows it is the correct number to use, and "blindly" use it, but if the caller pass `LibraryAccountType`, it might not be a number, and in this case, we explicitly "ask" if its an account id or library id, so if the manager need an account id `LibraryAccountType::LibraryId` will fail.